src/pages/index.vue

```diff
@@ -155,12 +167,18 @@ const HomePage = {
     }))
 
     const featuredSearchIdx = ref(-1) // immediately updates to 0 on mount
+    const featuredSearch = computed(
+      () =>
+        featuredSearches[
+          featuredSearchIdx.value >= 0 ? featuredSearchIdx.value : 0
+        ]
+    )
+
     const nextIdx = () => {
```

Comment:
@dhruvkb does this break the animate-in effect I think you were going for with the `-1`?